src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const array = [];
+  const cloneState = Object.assign({}, state);
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
Consider using the switch here

Reply:
<img width="771" alt="image" src="https://github.com/mate-academy/js_stateful-clones/assets/70220667/83042598-bdc6-400e-b89a-2c851c16927d">